### ISSUE TITLE
test

### ISSUE THREAD

**Issue by Vegetabledog77** [noahzn/Lite-Mono#181]
Dear Author,

Hello, I am a student who has just started learning about monocular depth estimation. Your work is truly outstanding. There is one part I don't quite understand: are all the visualized images in your paper from the KITTI test set (697 images)? Must the data used for visualization experiments come from the test set?

**Comment by noahzn** [comment]
Hi, yes, all the visualized images are from the test set. We are not supposed to use images from the training set for visualization.

**Comment by Vegetabledog77** [comment]
> Hi, yes, all the visualized images are from the test set. We are not supposed to use images from the training set for visualization.
Thank you for your quick reply. The results are acceptable when using pre-trained weights, but without pre-training, the performance is very poor. Do you know the reason for this?
Below is my training command:python train.py --data_path /home/m/shujuji/kitti_raw/kitti_raw_data  --model_name mytrain1 --num_epochs 30 --batch_size 12   --lr 0.0001 5e-6 31 0.0001 1e-5 31  --png 

![Image](https://github.com/user-attachments/assets/b9f2814f-e547-4637-99d5-3799b0fd8930)

**Comment by noahzn** [comment]
Hi, I'm afraid your learning rate is too small for a model trained from scratch. In the paper we used `5e−4`.

**Comment by Vegetabledog77** [comment]
> Hi, I'm afraid your learning rate is too small for a model trained from scratch. In the paper we used `5e−4`.

Thank you for your response. I will try again.
